### ISSUE TITLE
Issue with bulk delete or general bulk actions

### ISSUE THREAD

**Issue by callum-veloxcommerce** [akeneo/pim-community-dev#12309]
:bug: **I'm reporting a Bug** :bug:

I am on version 4.0.

Last night I upgraded from v3.2 to v4.0 and ever since I have started to receive issues with batch processing.

**For batch delete:**  
Expected behaviour:
> Select all products (roughly 26k) and click delete -> batch job starts and all products should be deleted

Actual behaviour:
> Select all products and click delete -> batch job starts and fails after 100 entries and logs following error: 
`Detached entity [4029989 cannot be removed`

**For batch edit:**
Expected behaviour:
> Select all products and click update family -> batch job starts and all products family changes

Actual behaviour:
> Select all products and click update family -> batch job starts and fails after 200 entries and logs the following errors:
```
An exception occurred while executing 'INSERT INTO pim_catalog_product (is_enabled, identifier, raw_values, created, updated, family_id, product_model_id, family_variant_id) VALUES (?, ?, ?, ?, ?, ?, ?, ?)' with params [1, "[4037420", "{\"sku\":{\"<all_channels>\":{\"<all_locales>\":\"[4037420\"}}}", "2020-06-16 14:58:16", "2020-06-16 14:58:16", 2, null, null]: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '[4037420' for key 'pim_catalog_product.UNIQ_91CD19C0772E836A'
--
FAILED The EntityManager is closed.

```

My system information is as follows:

```
[Edition]
CE

[Version]
4.0.33

[Environment]
prod

[Install time]
2020-06-15T21:45:40+0000

[Server version]
nginx/1.17.3

[Number of channels]
3

[Number of locales]
1

[Number of products]
0

[Number of product models]
0

[Number of variant products]
0

[Number of families]
2

[Number of attributes]
1

[Number of users]
4

[Number of categories]
5

[Number of category trees]
2

[Maximum number of categories in one category]
3

[Maximum number of category levels]
1

[Number of product values]
0

[Average number of product values by product]
0

[Average number of potential product values by family]
1

[Maximum number of potential product values by family]
1

[Associated email domains]
example.com,mylocalchemist.biz,mylocalchemist.com,veloxcommerce.co.uk,veloxcomnmerce.co.uk

[Number of scopable attributes]
0

[Number of localisable attributes]
0

[Number of localisable and scopable attributes]
0

[Number of attributes useable as grid filter]
1

[Average percentage of scopable attributes per family (%)]
0

[Average percentage of localisable attributes per family (%)]
0

[Average percentage of localisable and scopable attributes per family (%)]
0

[Average number of attributes per family]
1

[OS version]
Linux mlc-akeneo-demo 4.15.0-106-generic #107-Ubuntu SMP Thu Jun 4 11:27:52 UTC 2020 x86_64

[PHP version]
7.3.19-1+ubuntu18.04.1+deb.sury.org+1

[MySQL version]
8.0.20

[PHP extensions]
Core
date
libxml
openssl
pcre
zlib
filter
hash
Reflection
SPL
sodium
session
standard
cgi-fcgi
mysqlnd
PDO
xml
apcu
bcmath
calendar
ctype
curl
dom
mbstring
fileinfo
ftp
gd
gettext
iconv
igbinary
imagick
imap
intl
json
exif
msgpack
mysqli
pdo_mysql
pdo_pgsql
pdo_sqlite
pgsql
apc
posix
readline
redis
shmop
SimpleXML
soap
sockets
sqlite3
sysvmsg
sysvsem
sysvshm
tokenizer
wddx
xmlreader
xmlwriter
xsl
zip
Phar
memcached
Zend OPcache

[Registered bundles]
Akeneo\Channel\Bundle\AkeneoChannelBundle
Akeneo\Connectivity\Connection\Infrastructure\Symfony\AkeneoConnectivityConnectionBundle
Akeneo\Pim\Enrichment\Bundle\AkeneoPimEnrichmentBundle
Akeneo\Pim\Structure\Bundle\AkeneoPimStructureBundle
Akeneo\Platform\Bundle\AnalyticsBundle\PimAnalyticsBundle
Akeneo\Platform\Bundle\CatalogVolumeMonitoringBundle\PimCatalogVolumeMonitoringBundle
Akeneo\Platform\Bundle\DashboardBundle\PimDashboardBundle
Akeneo\Platform\Bundle\ImportExportBundle\PimImportExportBundle
Akeneo\Platform\Bundle\InstallerBundle\PimInstallerBundle
Akeneo\Platform\Bundle\NotificationBundle\PimNotificationBundle
Akeneo\Platform\Bundle\UIBundle\PimUIBundle
Akeneo\Tool\Bundle\ApiBundle\PimApiBundle
Akeneo\Tool\Bundle\BatchBundle\AkeneoBatchBundle
Akeneo\Tool\Bundle\BatchQueueBundle\AkeneoBatchQueueBundle
Akeneo\Tool\Bundle\ClassificationBundle\AkeneoClassificationBundle
Akeneo\Tool\Bundle\ConnectorBundle\PimConnectorBundle
Akeneo\Tool\Bundle\ElasticsearchBundle\AkeneoElasticsearchBundle
Akeneo\Tool\Bundle\FileStorageBundle\AkeneoFileStorageBundle
Akeneo\Tool\Bundle\MeasureBundle\AkeneoMeasureBundle
Akeneo\Tool\Bundle\StorageUtilsBundle\AkeneoStorageUtilsBundle
Akeneo\Tool\Bundle\VersioningBundle\AkeneoVersioningBundle
Akeneo\UserManagement\Bundle\PimUserBundle
Doctrine\Bundle\DoctrineBundle\DoctrineBundle
Doctrine\Bundle\DoctrineCacheBundle\DoctrineCacheBundle
Doctrine\Bundle\FixturesBundle\DoctrineFixturesBundle
Doctrine\Bundle\MigrationsBundle\DoctrineMigrationsBundle
FOS\JsRoutingBundle\FOSJsRoutingBundle
FOS\OAuthServerBundle\FOSOAuthServerBundle
FOS\RestBundle\FOSRestBundle
Liip\ImagineBundle\LiipImagineBundle
Oneup\FlysystemBundle\OneupFlysystemBundle
Oro\Bundle\ConfigBundle\OroConfigBundle
Oro\Bundle\DataGridBundle\OroDataGridBundle
Oro\Bundle\FilterBundle\OroFilterBundle
Oro\Bundle\PimDataGridBundle\PimDataGridBundle
Oro\Bundle\PimFilterBundle\PimFilterBundle
Oro\Bundle\SecurityBundle\OroSecurityBundle
Oro\Bundle\TranslationBundle\OroTranslationBundle
Sensio\Bundle\FrameworkExtraBundle\SensioFrameworkExtraBundle
Symfony\Bundle\AclBundle\AclBundle
Symfony\Bundle\FrameworkBundle\FrameworkBundle
Symfony\Bundle\MonologBundle\MonologBundle
Symfony\Bundle\SecurityBundle\SecurityBundle
Symfony\Bundle\SwiftmailerBundle\SwiftmailerBundle
Symfony\Bundle\TwigBundle\TwigBundle
```

**Comment by callum-veloxcommerce** [comment]
I am now experiencing even more bulk editing issues with the latest version of Akeneo:

Select 100 products and click Bulk edit:
![image](https://user-images.githubusercontent.com/53309514/85725343-ee9c9e00-b6ec-11ea-9f0f-00d12b2e955e.png)

A 500 server error occured:
![image](https://user-images.githubusercontent.com/53309514/85725446-04aa5e80-b6ed-11ea-8f94-a002a4efca1f.png)

No route found for "GET /rest/mass_edit/get-filter": Method Not Allowed (Allow: POST)
![image](https://user-images.githubusercontent.com/53309514/85725483-0f64f380-b6ed-11ea-9a6e-e7a8c275aa81.png)


All of this functionality worked perfectly prior to the upgrade, perhaps an upgrade step went wrong? any assistance is much appreciated.

**Comment by callum-veloxcommerce** [comment]
This was fixed by downgrading to Elastic Seach 7.5.2 and upgrading ghostscript to version 9.27.
